src/lightning_app/cli/lightning_cli.py

```diff
@@ -206,7 +208,24 @@ def stop():
     pass
 
 
-@_main.group(hidden=True)
+@stop.command("app")
+@click.argument("id", type=str)
+def stop_app(id: str):
```

Comment:
@awaelchli I could use some help with how to structure the logic for this command, specifically where to keep the API calls to the backend. Any suggestions?

Reply:
@alecmerdler I don't know if I'm the best to ask. As we can see looking at the file structure in the cli folder, it is already a mess. Perhaps it is best to follow the pattern that emerged with the other commands for create, delete, list, etc. 

`lightning_cli_stop.py` contains the stop CLI subcommand definition
`cmd_stop.py` contains the api calls, error handling etc. 

What do you think?
cc @tchaton

Reply:
I agree, the CLI could use some cleanup. Your suggestion makes sense as a basic step to separate logic from presentation. I'll change the code to do that.